lib/compiler/src/target.rs

```diff
@@ -160,9 +161,11 @@ impl ToString for CpuFeature {
 
 /// This is the target that we will use for compiling
 /// the WebAssembly ModuleInfo, and then run it.
-#[derive(Clone, Debug, PartialEq, Eq, Hash)]
+#[derive(Clone, Debug, PartialEq, Eq, Hash, MemoryUsage)]
 pub struct Target {
+    #[loupe(skip)]
     triple: Triple,
+    #[loupe(skip)]
     cpu_features: EnumSet<CpuFeature>,
 }
```

Comment:
This might be the same as the thing I mentioned above, not high priority and easy to add later, but might be able to do something like

```
#[loupe(skip_all)]
pub struct Target {
...
}
```

To just get the local size and not any of the heap size stuff

Reply:
I've created https://github.com/wasmerio/loupe/issues/12.